plugin.yml

```
@@ -9,6 +9,8 @@ configuration:
       type: string
     name:
       type: string
+    scan-on-push:
+      type: string
```

Should we treat this as a boolean?

```suggestion
      type: boolean
```

Here's an example of matching a YAML _true_: https://github.com/buildkite-plugins/docker-buildkite-plugin/blob/5745e8f7f3cc99343b25bb57381288192df3331c/hooks/command#L90

---

We should update this to boolean:

https://github.com/seek-oss/create-ecr-buildkite-plugin/blob/d16f8bc2c1db595a5afc29f30575aeb945cf7c8c/README.md#L40

And this to check for the many trues in YAML per the link above:

https://github.com/seek-oss/create-ecr-buildkite-plugin/blob/d16f8bc2c1db595a5afc29f30575aeb945cf7c8c/hooks/pre-command#L41

---

Both comments addressed.